SerialPrograms/Source/PokemonSV/Programs/Farming/PokemonSV_MaterialFarmerTools.cpp

```diff
@@ -3,7 +3,7 @@
  *  From: https://github.com/PokemonAutomation/Arduino-Source
  *
  */
-
+#include <QDir>
```

Comment:
Why is this needed? We try to keep Qt out of the main programs, though some exceptions are allowed for things that we don't have our own implementation for. (stuff that we should go back an cleanup at some point)

Reply:
My bad. It was from when I was experimenting with dumping screenshots. It's not needed. I removed it now.